KIPs/kip-34.md

```diff
@@ -476,9 +481,28 @@ None
 | createWithRoleBasedKey(address: String, roleBasedKeys: List&#60;String[]&#62;): RoleBasedKeyring | Creates a RoleBasedKeyring instance from an address and a 2D array of which each array element contains keys defined for each role. It throws an exception if the address string or the private key strings are invalid. |
 | decrypt(keystore: Object, password: String): Keyring | Decrypts a keystore v3 or v4 JSON and returns the decrypted keyring instance. It throws an exception if the decryption is failed. |
 
+#### IWallet
+
+`IWallet` is the interface of Wallet that manages data used when signing transactions. All Wallet classes must implement `IWallet`.
+Caver has a `KeyringContainer` instance that implements `IWallet`, and if it is a Wallet class that implements `IWallet` interface, it can be used in Caver instead of `KeyringContainer`.
+
+##### Variables
+
+None
+
+##### Methods
+
+| Method | Description |
+| ----------- | ----------- |
+| generate(num: int): List<String> | Generates `num` SingleKeyring instances and stores them in the `Wallet` instance. |
+| sign(address: String, transaction: AbstractTransaction): AbstractTransaction | Signs the transaction as a sender of the transaction and appends signatures in the transaction object using the account associated with the given address in the `Wallet` instance. It throws an exception if the account to be used for signing cannot be found in the `Wallet` instance or if `address` is different from the sender address of `transaction`. |
+| signAsFeePayer(address: String, transaction: AbstractFeeDelegatedTransaction): AbstractFeeDelegatedTransaction | Signs the transaction as a fee payer of the transaction and appends signatures in the transaction object using the account associated with the given address in the `Wallet` instance. It throws an exception if the account to be used for signing cannot be found in the `Wallet` instance or if the address received as a parameter and the fee payer of the actual transaction do not match. |
+| isExisted(address: String): Boolean | Returns whether the account corresponding to the address exists. |
+| remove(address: String): Boolean | Deletes an account associated with the given address from the `Wallet` instance. |
```

Comment:
```suggestion
| remove(address: String): Boolean | Deletes an account associated with the given address. |
```

Reply:
I applied in here https://github.com/klaytn/kips/pull/53/commits/6b1c2acde7f0aa1970bfa8aecd0e018cb3f31504